build_poh.sh

```diff
@@ -29,19 +31,18 @@ do
     fi
 done
 
-set -e
+# get the Open Hospital version
+version=$(grep -m 1 "</version>" pom.xml | grep -o "[0-9.]*")
```

Comment:
not sure of this: we trigger the release with a tag, that should be available at the build time and should be the same as in the other repositories, isn't it?

Reply:
yes this repo gets tagged before releasing and the tag should be the same as in the other repos, and the same as in that pom.xml.  
You mean: we can use the tag instead of reading the version from pom.xml? If so, yes, that can be done as well I suppose. I think it's more or less the same.

Reply:
> You mean: we can use the tag instead of reading the version from pom.xml?

Yes, I mean that. 

Also, is pom.xml really needed in this repo?

Reply:
> Yes, I mean that.

Alright, I'll try to do that in the next days

> Also, is pom.xml really needed in this repo?

it is used to compile core and gui as Maven submodules: https://github.com/informatici/openhospital-core/blob/develop/pom.xml#L29
It can be avoided, but then instead of a simple `mvn package` (as [here](https://github.com/informatici/openhospital-core/blob/develop/pom.xml#L29)) you should go into each project dir, compile it, change dir, etc. All feasible.
Maybe you can open an issue and we can remove it in another pr?